backend/apps/progress/views.py

```
@@ -67,154 +66,30 @@ def get(self, request):
-
-        multiplier = XPMultiplierEvent.get_active_multiplier()
         base_score = request.data.get("score", 100)
         completed = request.data.get("completed", True)
 
         try:
-            lesson = Lesson.objects.get(
-                slug=lesson_slug,
-                organization=request.user.organization,
-            )
-        except Lesson.DoesNotExist:
+            progress, created, idempotency_hit = ProgressTrackingService.record_lesson_progress(
```

Ruff (0.15.20)

[warning] Unpacked variable `idempotency_hit` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)